source/local/local.go

```diff
@@ -144,17 +163,57 @@ func (s *localSource) GetFeatures() *nfdv1alpha1.Features {
 	return s.features
 }
 
-func parseFeatures(lines [][]byte) map[string]string {
+func parseDirectives(line string, fileName string, opts *parsingOpts) error {
+	lineSplit := strings.SplitN(line, "=", 2)
+
+	// If we have the following directive: # +expiry-time=xxx
+	// we need to get the key '# +expiry-time' and then we need to extract
+	// the following key 'expiry-time'.
+	key := strings.Split(lineSplit[0], DirectivePrefix)[1]
+	if len(lineSplit) == 1 {
+		return ErrorWrongDirectiveFormat
+	}
+	value := lineSplit[1]
+
+	switch key {
+	case ExpiryTimeKey:
+		expiryDate, err := time.Parse(time.RFC3339, strings.TrimSpace(value))
+		if err != nil {
+			klog.ErrorS(err, "failed to parse feature file expiry date", "fileName", fileName)
+			return err
+		}
+		opts.ExpiryTime = expiryDate
+	default:
+		klog.ErrorS(ErrorUnknownDirective, "unknown directive", "directiveKey", key)
+		return ErrorUnknownDirective
```

Comment:
Same here

```suggestion
		return fmt.Errorf("unknown feature file directive %q", key)
```

...or something